tools/link.py

```diff
@@ -2390,35 +2390,30 @@ def modularize():
     diagnostics.warning('emcc', 'EXPORT_NAME should not be named "config" when targeting Safari')
 
   if settings.MODULARIZE == 'instance':
-    src = '''
+    wrapper_funcion = '''
 export default async function init(moduleArg = {}) {
   var moduleRtn;
 
-%(src)s
+%(generated_js)s
 
   return await moduleRtn;
 }
-''' % {
-      'src': src,
-    }
+''' % {'generated_js': generated_js}
   else:
-    src = '''
+    wrapper_funcion = '''
 %(maybe_async)sfunction(moduleArg = {}) {
   var moduleRtn;
 
-%(src)s
+%(generated_js)s
 
   return moduleRtn;
 }
-''' % {
-      'maybe_async': async_emit,
-      'src': src,
-    }
+''' % {'maybe_async': async_emit, 'generated_js': generated_js}
 
   if settings.MINIMAL_RUNTIME and not settings.PTHREADS:
     # Single threaded MINIMAL_RUNTIME programs do not need access to
     # document.currentScript, so a simple export declaration is enough.
-    src = '/** @nocollapse */ var %s = %s' % (settings.EXPORT_NAME, src)
+    src = f'/** @nocollapse */ var {settings.EXPORT_NAME} = {wrapper_funcion};'
```

Comment:
is the added `;` intentional?

Reply:
Yes, I think it was simply missing before.